File: medplum-java/medplum-model/src/main/java/com/fhir/ExplanationOfBenefit_Payee.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableExplanationOfBenefit_Payee.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface ExplanationOfBenefit_Payee {
  Optional<List<Extension>> extension();

  Optional<List<Extension>> modifierExtension();

  Optional<Reference> party();

  Optional<CodeableConcept> type();

  Optional<String> id();

  static ImmutableExplanationOfBenefit_Payee.Builder builder() {
    return ImmutableExplanationOfBenefit_Payee.builder();
  }
}

File: medplum-typescript/src/model/fhir/ExplanationOfBenefit_Payee.ts
import { CodeableConcept } from "./CodeableConcept";
import { Extension } from "./Extension";
import { Reference } from "./Reference";

export interface ExplanationOfBenefit_Payee {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    type: CodeableConcept | null | undefined;
    party: Reference | null | undefined;
}
